.github/workflows/notify-release.yml

```diff
@@ -0,0 +1,26 @@
+name: PRs not yet released checker
+on:
+  workflow_dispatch:
+  schedule:
+    # Runs on Mondays at 8AM PST (15 UTC)
+    - cron: '0 15 * * 1'
+
+jobs:
+  checking-pending-prs:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v2
+    - name: Use Node.js ${{ matrix.node-version }}
+      uses: actions/setup-node@v2
+      with:
+        node-version: lts/*
+    - name: Install Dependencies
+      run: npm ci
+    - name: Check for PRs not yet released
+      run: node ./bin/pending-prs.js
+      env:
+        GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+        SLACK_CHANNEL: ${{ secrets.SLACK_CHANNEL }}
+        SLACK_TOKEN: ${{ secrets.SLACK_TOKEN }}
+        SLACK_SECRET: ${{ secrets.SLACK_SECRET }}
```

Comment:
You prob have already but please make sure these are stored in the standard place for team retrieval/recovery in the future.

Reply:
you mean put into last pass? I did document in https://docs.google.com/document/d/1mPSYY9aJjyah908oum71jlqXhwo-PZ2n65wynizwIQk/edit where to find these

Reply:
yeah, that's where i look for credentials naturally. glad where to find things is documented although i'm guessing i'll forget that document exists in a few months.